tools/ReleaseNotes/data/whats-new-95.md

```diff
@@ -41,16 +41,19 @@ ms.date: 08/21/2025
   - [OpenAI hosting integration](#openai-hosting-integration)
   - [GitHub Models typed catalog](#github-models-typed-catalog)
   - [Dev Tunnels hosting integration](#dev-tunnels-hosting-integration)
+  - [YARP static files support](#yarp-static-files-support)
+  - [Redis and RabbitMQ auto activation](#redis-and-rabbitmq-auto-activation)
```

Comment:
We should add [Add support for Azure Redis enterprise (dotnet/aspire#10726)](https://github.com/dotnet/aspire/pull/10726)